compiler/testData/diagnostics/tests/dataFlowInfoTraversal/UnaryExpression.fir.kt

```diff
@@ -4,11 +4,11 @@ fun baz(b: Boolean): Boolean = !b
 fun foo() {
     val x: Int? = null
 
-    bar(<!UNSAFE_CALL!>-<!>x)
+    bar(<!UNSAFE_CALL!>-x<!>)
```

Comment:
Hm, not sure how to interpret this change (v.s. https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/dataFlowInfoTraversal/UnaryExpression.kt#L7)

Reply:
Also reverted to the old behavior.